pool/src/lib.rs

```diff
@@ -32,7 +32,8 @@ extern crate grin_keychain as keychain;
 extern crate grin_util as util;
 extern crate rand;
 extern crate serde;
-#[macro_use]
+#[allow(unused_imports)]
+#[macro_use] // Needed for Serialize/Deserialize. The compiler complaining here is a bug.
```

Comment:
Can we do the `#[cfg(test)]` trick here?

Reply:
No, Serialize/Deserialize is used in a --release build, and I couldn't come up with any combination of allow(unused_imports) and/or cfg[test] to silence the compiler complaints here -- that's why the commit message is "Tried but failed" and then hoping that not all of us (maybe repeatedly) will loose time fighting that particular warning for this particular lib. I think I tried fixing this once before, but found no commits to jog my memory...